udata/templates/admin.html

```diff
@@ -10,18 +10,12 @@
             'robots': 'noindex',
         }) }}
         <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport' />
-        {% if 'common.css' is in_manifest %}
-        <link href="{{ manifest('udata', 'common.css') }}" rel="stylesheet" />
-        {% endif %}
-        <link href="{{ manifest('udata', 'admin.css') }}" rel="stylesheet" />
-        {% block theme_css %}{% endblock %}
-        {% block extra_head %}{% endblock %}
+        <link href="{{ static('admin.css') }}" rel="stylesheet" />
```

Comment:
I think we can keep these blocks, that could be extended by a theme (we should dive in this later).
```suggestion
        <link href="{{ static('admin.css') }}" rel="stylesheet" />
        {% block theme_css %}{% endblock %}
        {% block extra_head %}{% endblock %}
```

Reply:
the admin does not use the theme and `udata` is not suppose to be aware of theme logic as it was moved to `udata-front`